web/src/recent_view_util.ts

```diff
@@ -1,15 +1,33 @@
 import type {Message} from "./types";
 
 let is_rt_visible = false;
+let rt_stream_id: number | null = null;
```

Comment:
Let's call this `current_stream_id`; and maybe also rename `is_rt_visible` to `is_visible` in a prep commit; the `rt` is an abbreviation and thus undesired, and also just unnecessary given what module this is.

Reply:
I decided to use `is_view_visible` since `is_visible` is used as the name of a function, and renamed to `current_stream_id` as you described